tests/cypress/support/step_definitions/jupyterlite/I see output in the cell # is.ts

```diff
@@ -0,0 +1,17 @@
+import { Then } from "@badeball/cypress-cucumber-preprocessor";
+import MaterialDesignerPage from "../../widgets/MaterialDesignerPage";
+
+function normalizeText(code: string) {
+    return code.replace(/\s/g, "");
```

Comment:
Should be in utils? What exactly is `normalize` here - do we just strip out the whitespace?